charts/cofide-connect/templates/configmap-envoy.yaml

```diff
@@ -146,7 +146,7 @@ data:
                           '@type': type.googleapis.com/envoy.extensions.filters.http.jwt_authn.v3.JwtAuthentication
                           providers:
                             ory-oauth:
-                              issuer: {{ .Values.envoy.auth.issuer }}
+                              issuer: https://{{ .Values.envoy.auth.issuer }}
```

Comment:
Should the issuer value be a URL, with the host extracted using urlParse if necessary?

Reply:
I've only now realised what you mean, sorry. Changed